src/morphology_workflows/repair.py

```diff
@@ -319,71 +321,96 @@ def make_collage(
                 plt.close()
 
 
-def make_release(  # pylint: disable=unused-argument
-    df, data_dir, zero_diameter_path=None, unravel_path=None, repair_path=None
-):
-    """Make morphology release."""
-    if zero_diameter_path is not None:
-        zero_diameter_path = Path(zero_diameter_path)
-        zero_diameter_path.mkdir(exist_ok=True, parents=True)
-
-    if unravel_path is not None:
-        unravel_path = Path(unravel_path)
-        unravel_path.mkdir(exist_ok=True, parents=True)
-
-    if repair_path is not None:
-        repair_path = Path(repair_path)
-        repair_path.mkdir(exist_ok=True, parents=True)
-
-    _m = []
-    for name in df.loc[df["is_valid"]].index:
-        if "mtype" in df.columns and isinstance(df.loc[name, "mtype"], str):
-            mtype = df.loc[name, "mtype"]
-            layer = df.loc[name, "mtype"][1]
-        else:
-            mtype = ""
-            layer = ""
-        _m.append(
-            MorphInfo(
-                name=name,
-                mtype=mtype,
-                layer=layer,
-                use_dendrite=df.loc[name, "has_basal"],
-                use_axon=df.loc[name, "has_axon"],
-            )
-        )
+def _convert(input_file, output_file):
+    """Handles crashes in convertion of writing of morphologies."""
+    try:
+        convert(input_file, output_file)
+        return output_file
+    except MorphToolException:
+        return "cannot save"
+
+
+def _get_layer_mtype(mtype_input):
+    """Helper to get layer from mtype, if mtype exists as a str."""
+    layer = "no_layer"
+    mtype = "no_mtype"
+    if isinstance(mtype_input, str):
+        mtype = mtype_input
+        if len(mtype_input) > 1:
+            layer = mtype_input[1]
+    return mtype, layer
+
 
+def make_release(df, _, zero_diameter_path, unravel_path, repair_path, extensions):
+    """Make morphology release."""
+    for extension in extensions:
         if zero_diameter_path is not None:
-            zero_diameter_release_path = (
-                zero_diameter_path / Path(df.loc[name, "zero_diameter_morph_path"]).name
+            _zero_diameter_path = Path(f"{zero_diameter_path}-{extension[1:]}")
+            _zero_diameter_path.mkdir(exist_ok=True, parents=True)
+
+        if unravel_path is not None:
+            _unravel_path = Path(f"{unravel_path}-{extension[1:]}")
+            _unravel_path.mkdir(exist_ok=True, parents=True)
+
+        if repair_path is not None:
+            _repair_path = Path(f"{repair_path}-{extension[1:]}")
+            _repair_path.mkdir(exist_ok=True, parents=True)
+
+        _m = []
+        for name in df.loc[df["is_valid"]].index:
```

Comment:
Let's hope the DF is always small... ;-P
If you can vectorize this operation it would be nice.